programs/bpf/c/sdk/inc/stdio.h

```diff
@@ -0,0 +1,2 @@
+#pragma once
+typedef void *FILE;
```

Comment:
Where are you headed with *FILE?

Reply:
Nowhere!  There was just a function that wanted a FILE as input in the generated bindings.  We would never use such a function, but the compiler barfs without the definition.

Reply:
Maybe we should make it explicit that we are not planning to support *FILE by putting its declaration in another Solana header (solana_not_supported.h (better name needed))